packages/python/patch-tm_zone

```diff
@@ -0,0 +1,14 @@
+--- Modules/timemodule.c
```

Comment:
It's quite annoying that this is needed. Cloudlibc declares the non-standard `tm_zone` different from other C libraries. In our case it's an array instead of a pointer. The reason for this is that time zone strings are generated on the spot by `localtime_l()`. If we want it to be a pointer, we need to change it to use a string pool internally. Blegh.